projects/js-packages/licensing/components/activation-screen-error/style.scss

```diff
@@ -0,0 +1,63 @@
+.activation-screen-error__message {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    max-width: 500px;
+    margin-top: calc(var(--spacing-base)*.5);
+
+    svg {
+        margin-left: -3px;
+    }
+
+    span {
+        font-size: 13px;
+        line-height: 20px;
+        font-weight: 500;
+        letter-spacing: -0.044em;
+    }
+}
+
+.activation-screen-error__message--error {
+    color: var(--jp-red);
+    svg {
+        fill: var(--jp-red);
+    }
+}
+
+.activation-screen-error__message--success {
+    color: var(--jp-green);
+    svg {
+        fill: var(--jp-green);
+    }
+}
+
+.activation-screen-error__info {
+    background-color: var(--jp-gray-0);
+    color: var(--jp-gray-80);
+    font-size: var(--font-body-small);
+    line-height: calc(var(--font-title-small) - 2px);
+    border: 1px solid #F0F0F0;
+    border-radius: var(--jp-border-radius);
+    padding: var(--jp-modal-padding-small);
+    margin: 32px 0 8px;
+
+    > p {
+        margin: 0 0 1em !important;
```

Comment:
Wonder if it may be possible to get rid of these `! important` tags if you added a bit of specificity to the `.activation-screen-error__info` class above. Maybe changing it to `div.activation-screen-error__info` would work? Even if I add a new CSS class selector via the browser inspector it will always override the current styles so I couldn't test this without booting up my local environment